### DISCUSSION TITLE
README.md has stopped committing changes.

### DISCUSSION THREAD

**Discussion by AAdewunmi** [community/community#15677] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi, I made some changes to a README.md file and tried to commit my changes by clicking on the green button save.
Unfortunately, the process seems to have stalled; the green button has been showing the message "Saving ..." for the 
past 10 minutes.
Any suggestions on a fix?
Thanks.  
![Screenshot 2022-04-27 at 14 02 05](https://user-images.githubusercontent.com/15172744/165524003-f6529d4b-859d-48c3-a04c-9f02dc2d6aeb.png)

**Comment by queenofcorgis** [comment, marked as answer]
Hi there - are you still experiencing this?

**Reply by AAdewunmi** [reply]
Hi, it's working. Thanks 👍
